README.md

```diff
@@ -1 +1,10 @@
 # hw01
+## Omair Gill's Biography
+I was born and raised in *San Mateo, California*. I attended my undergraduate studies at University of California, Davis and double majored in **Economics and International Relations**. I am currently a second year MPP student at the Harris School of Public Policy. Please see [my Linkedin profile] (https://www.linkedin.com/in/omair-gill-1b0038164) for more information on my professional background. My hobbies include:  
```

Comment:
Adding the link to your linkedin profile was a good touch!

Reply:
Thanks Sharnaya! I thought it was supposed to look like that...I didn't realize the goal was to link the text to a link, lol.

Reply:
@ogill-stuart if you remove the space, it will format appropriately

```
[my Linkedin profile](https://www.linkedin.com/in/omair-gill-1b0038164)
```

[my Linkedin profile](https://www.linkedin.com/in/omair-gill-1b0038164)